web/src/lib/components/shared-components/version-announcement-box.svelte

```diff
@@ -1,12 +1,13 @@
 <script lang="ts">
-  import { getGithubVersion } from '$lib/utils/get-github-version';
   import { onMount } from 'svelte';
   import FullScreenModal from './full-screen-modal.svelte';
-  import type { ServerVersionReponseDto } from '@api';
+  import { api, type ServerVersionReponseDto } from '@api';
   import Button from '../elements/buttons/button.svelte';
 
   export let serverVersion: ServerVersionReponseDto;
 
+  const disable_check: boolean = import.meta.env.CHECK_NEW_VERSION_INTERVAL === '0';
```

Comment:
Does this work? I don't think non PUBLIC_* environment variables are available in this context. Also, can't the logic to skip the check happen in the server? Like it can ignore the update check and always return false. I don't think we need to have a check client side and a check server side for this feature.